app/celery/scheduled_tasks.py

```diff
@@ -533,3 +535,29 @@ def weekly_dwp_report():
         ),
         due_at=convert_utc_to_bst(datetime.utcnow()) + timedelta(days=7),
     )
+
+
+@notify_celery.task(bind=True, name="change-dvla-password", max_retries=55, retry_backoff=True, retry_backoff_max=300)
```

Comment:
Yeah I agree this is probably excessive and might just interfere with us manually getting involved. Maybe just try 3 times in total and then fail and bubble up for manual intervention?